lib/bundle/bundle.rb

```diff
@@ -25,6 +25,10 @@ def mas_installed?
       @mas_installed ||= which_formula("mas")
     end
 
+    def vscode_installed?
+      @vscode_installed ||= which("code").present?
```

Comment:
Will consider this as a follow-up PR if someone can test and verify this all works as expected for all commands and with all extensions.

Reply:
this will also be the same for vscode insiders builds. it creates a `code-insiders` executable as opposed to `code`. should we have a look up for the known variants and use which ever we find in the `PATH` first?

Reply:
Same deal: if someone using these tests it and opens a PR: will merge.